packages/apps-config/src/api/index.ts

```diff
@@ -1,10 +1,7 @@
 // Copyright 2017-2023 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { applyDerives } from './derives.js';
-import { typesBundle as typesOnlyBundle } from './typesBundle.js';
+export { typesBundle } from './typesBundle.js';
 
 export * from './params/index.js';
 export * from './constants.js';
-
-export const typesBundle = applyDerives(typesOnlyBundle);
```

Comment:
From what I understand, derives relate only to [some chains](https://github.com/Cardinal-Cryptography/apps/pull/74/files#diff-a5facc6227c8d5c5461d1550e9112093a18f9e94be4d74bd5fda8439bdfe0416L13-L17) - not to A0.